src/app/src/index.js

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { render } from 'react-dom';
+import { Provider } from 'react-redux';
+
+import * as serviceWorker from './serviceWorker';
```

Comment:
Since this project is going to be hosted locally from the machine the browser is on, I don't think we get any performance benefits from using a service worker for caching. On the other hand, service workers can make development more frustrating, so we should consider dropping it entirely.

That said, this looks like it came from the project template, not this PR, so this comment doesn't need to be addressed now, we could push it off to a separate issue.

Reply:
🦅 👁 
Yeah, the service worker comes as part of the Create React App. It is, by default, not registered.